syntax = "proto3";
option java_package = "com.alipay.rpc.common.service.facade.pb.request";
option java_multiple_files = true;
option go_package = "com.alipay.rpc.common.service.facade.pb";
import "SubServicePbRequest.proto";
message ComplexServicePbRequest {
    string name = 1;
    SubServicePbRequest subServicePbRequest = 2;
}

